Content.Server/Weapon/Ranged/Ammunition/Components/AmmoComponent.cs

```diff
@@ -82,16 +81,15 @@ public bool Spent
         [DataField("ammoVelocity")]
         public float Velocity { get; } = 20f;
 
-        [DataField("muzzleFlash")]
-        private string _muzzleFlashSprite = "Objects/Weapons/Guns/Projectiles/bullet_muzzle.png";
+        [DataField("muzzleFlash")] public string? MuzzleFlashSprite = "Objects/Weapons/Guns/Projectiles/bullet_muzzle.png";
```

Comment:
Is there a funny serv3 thing for this now?

Reply:
`ResourcePathSerializer` works I think

Reply:
you will have to, yes